File: trunk/Lib/DotNet/aaaSoft.Net/aaaSoft.Net/Xiep/EventArgs/XiepClientEventArgs.cs
using System;
using System.Collections.Generic;
using System.Text;
using aaaSoft.Net.Xiep.Packages;

namespace aaaSoft.Net.Xiep.EventArgs
{
    public class XiepClientEventArgs : System.EventArgs
    {
        private EventPackage eventPackage;

        //获取事件包
        public EventPackage getEventPackage()
        {
            return eventPackage;
        }

        //构造函数
        public XiepClientEventArgs(EventPackage eventPackage)
        {
            this.eventPackage = eventPackage;
        }
    }
}

File: trunk/Lib/Java/com.scbeta.net/src/com/scbeta/net/xiep/eventArgs/XiepClientEventArgs.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.scbeta.net.xiep.eventArgs;

import com.scbeta.net.xiep.packages.EventPackage;
import java.util.EventObject;

/**
 * XiepClient事件参数类
 * @author aaa
 */
public class XiepClientEventArgs extends EventObject {

    private EventPackage eventPackage;

    //获取事件包
    public EventPackage getEventPackage() {
        return eventPackage;
    }

    //构造函数
    public XiepClientEventArgs(Object source, EventPackage eventPackage) {
        super(source);
        this.eventPackage = eventPackage;
    }
}
